templates/monitoring/kube_state_metrics_3scale_alerts.yaml

```diff
@@ -94,6 +94,24 @@ spec:
         for: 5m
         labels:
           severity: critical
+      - alert: ThreeScaleZyncPodAvailability
+        annotations:
+          message:  3Scale Zync has no pods in a ready state.
```

Comment:
This message is not completely accurate. The alert will trigger if there not exactly 2 zync pods in a ready state.

Reply:
:+1: